locales/hu_HU/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,775 @@
+
+msgid "1 – 1.5kV ="
+msgstr ""
+
+msgid "block signal type So"
+msgstr ""
+
+msgid "%SPDMIN%-%SPDMAX% km/h"
+msgstr ""
+
+msgid "1.5kV ="
+msgstr ""
+
+msgid "El 2 Einschaltsignal"
+msgstr ""
+
+msgid "Ra 10 Rangierhalttafel / Verschubhalttafel"
+msgstr ""
+
+msgid "Zs 10 Endesignal (Tafel)"
+msgstr ""
+
+msgid "El 1 Ausschaltsignal"
+msgstr ""
+
+msgid "Ra 11b Wartezeichen"
+msgstr ""
+
+msgid "El 6 Halt für Fahrzeuge mit gehobenem Stromabnehmer"
+msgstr ""
+
+msgid "Less than 750V ="
+msgstr ""
+
+msgid "Punktförmige Zugbeeinflussung"
+msgstr ""
+
+msgid "Operating Sites"
+msgstr ""
+
+msgid "train protection"
+msgstr ""
+
+msgid "contact line"
+msgstr ""
+
+msgid "El 4 \"Bügel ab\"-Signal"
+msgstr ""
+
+msgid "750V ="
+msgstr ""
+
+msgid "25kV 60Hz ~"
+msgstr ""
+
+msgid "El 3 \"Bügel ab\"-Ankündigungssignal"
+msgstr ""
+
+msgid "Electrification proposed"
+msgstr ""
+
+msgid "El 1v Ausschaltsignal erwarten"
+msgstr ""
+
+msgid "15kV 16.7Hz ~"
+msgstr ""
+
+msgid "Sh-Lichtsperrsignal (normal, Zwergsignal)"
+msgstr ""
+
+msgid "distant signal with Eo 2 (new type)"
+msgstr ""
+
+msgid "Ks-Hauptsignal"
+msgstr ""
+
+msgid "Bü 5 Läutetafel (normal, für durchfahrende Züge)"
+msgstr ""
+
+msgid "Ks-Mehrabschnittssignal (normal, verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "Vr-Lichtvorsignal mit Vr 2 (normal, Wiederholer/verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "Bü 2 Rautentafel (normal, verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "Ks-Vorsignal (normal, Wiederholer, verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "Bü 4 Pfeiftafel (normal, für durchfahrende Züge)"
+msgstr ""
+
+msgid "ex-Pf 1 Pfeiftafel (normal, für durchfahrende Züge)"
+msgstr ""
+
+msgid "El 4 \"Bügel ab\"-Signal"
+msgstr ""
+
+msgid "Vr-Formvorsignal ohne Angabe der Signalzustände"
+msgstr ""
+
+msgid "Vr-Formvorsignal (ohne Vr 2, mit Vr 2)"
+msgstr ""
+
+msgid "main signal without aspects given (new type, old type)"
+msgstr ""
+
+msgid "Hp-Lichthauptsignal (ohne Hp 2, mit Hp 2)"
+msgstr ""
+
+msgid "main signal with Po 1 (new type, old type)"
+msgstr ""
+
+msgid "main signal with Po 2 (new type, old type)"
+msgstr ""
+
+msgid "El 3 \"Bügel ab\"-Ankündigungssignal"
+msgstr ""
+
+msgid "distant signal with Eo 1 (new type, old type)"
+msgstr ""
+
+msgid "Vr-Lichtvorsignal ohne Vr 2 (normal, Wiederholer/verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "Sh-Formsperrsignal (normal, Zwergsignal)"
+msgstr ""
+
+msgid "Sv-Signal (ohne Hp 0, mit Hp 0)"
+msgstr ""
+
+msgid "Hp-Formhauptsignal (ohne Hp 2, mit Hp 2)"
+msgstr ""
+
+msgid "El 5 \"Bügel an\"-Signal"
+msgstr ""
+
+msgid "Vr-Lichtvorsignal ohne Angabe der Signalzustände (normal, Wiederholer/verkürzter Bremswegabstand)"
+msgstr ""
+
+msgid "distant signal without aspects given (new type, old type)"
+msgstr ""
+
+msgid "Zs 3v Geschwindigkeits­voranzeiger (Tafel, Lichtsignal)"
+msgstr ""
+
+msgid "Zs 3 Geschwindigkeits­anzeiger (Tafel, Lichtsignal)"
+msgstr ""
+
+msgid "Imprint &amp; Privacy Policy"
+msgstr ""
```

Comment:
This one is probably important. Looks otherwise fine.

Reply:
I added the translation of this string.